internal/pkg/image_repositories/template/Dockerfile

```diff
@@ -31,4 +31,10 @@ RUN <<EOF /bin/bash
 EOF
 {{- else if eq .Project "magnum" }}
 COPY --from=gcr.io/go-containerregistry/crane /ko-app/crane /usr/local/bin/crane
+RUN <<EOF /bin/bash
+  curl -Lo /tmp/helm.tar.gz "https://get.helm.sh/helm-v3.10.3-linux-amd64.tar.gz"
+  tar -zxvf /tmp/helm.tar.gz -C /tmp
+  mv /tmp/linux-amd64/helm /usr/local/bin/helm
+  rm -rf /tmp/helm.tar.gz /tmp/linux-amd64/
+EOF
```

Comment:
Is there no official Helm image that we can just copy directly from like we do in Crane?

Reply:
unfortunately, i couldn't find it in dockerhub and quay.io.
Maybe we can use `hub.docker.com/r/alpine/helm` ?

Reply:
that one works, because it seems like it's just copying the binary:

https://github.com/alpine-docker/helm/blob/master/Dockerfile#L20-L23